tools/load_demo2.py

```diff
@@ -12,6 +12,17 @@
 if(len(sys.argv) > 2):
     port = sys.argv[2]
 
+user = "postgres"
+password = "mamikonyan"
```

Comment:
Can we change the default password to something like 'postgres' or 'admin'? Thanks!